diff --git a/processable/coreschemas/NarrativeBlock.xsd b/processable/coreschemas/NarrativeBlock.xsd
index ebe1d11..a45a94e 100644
--- a/processable/coreschemas/NarrativeBlock.xsd
+++ b/processable/coreschemas/NarrativeBlock.xsd
@@ -45,11 +45,7 @@
   <!-- DELETE THIS, we don't need to define a global element for text
 	<xs:element name="text" type="text"/>
  -->
-  <xs:simpleType name="StrucDoc.Br">
-    <xs:restriction base="xs:string">
-      <xs:maxLength value="0"/>
-    </xs:restriction>
-  </xs:simpleType>
+  <xs:complexType name="StrucDoc.Br"/>
   <xs:complexType name="StrucDoc.Caption" mixed="true">
     <xs:choice minOccurs="0" maxOccurs="unbounded">
       <xs:element name="linkHtml" type="StrucDoc.LinkHtml"/>