diff --git a/infrastructure/cda/POCD_MT000040_extensions.xsd b/infrastructure/cda/POCD_MT000040_extensions.xsd
index 1bf750b..1b484bc 100644
--- a/infrastructure/cda/POCD_MT000040_extensions.xsd
+++ b/infrastructure/cda/POCD_MT000040_extensions.xsd
@@ -58,9 +58,9 @@ StaticMifToXsd.xsl version 2.0</xs:documentation>
 		schemaLocation="../../processable/extensions/ihelab.xsd"/>
 	<!-- Added SDTC namespace which holds the SDTC extensions -->
 	<xs:import namespace="urn:hl7-org:sdtc" schemaLocation="SDTC.xsd"/>
-	<xs:import namespace="urn:ihe:pharm:medication"
-		schemaLocation="COCT_MT230100UV_ext_pharmacy.xsd"/>
-	<xs:import namespace="urn:dicom-org:ps3-20" schemaLocation="DICOM.xsd"/>
+	<!-- <xs:import namespace="urn:ihe:pharm:medication"
+		schemaLocation="COCT_MT230100UV_ext_pharmacy.xsd"/> -->
+	<!-- <xs:import namespace="urn:dicom-org:ps3-20" schemaLocation="DICOM.xsd"/> -->

 	<xs:complexType name="POCD_MT000040.InfrastructureRoot.typeId">
 		<xs:complexContent>
